public/form/login.html

```diff
@@ -2,34 +2,64 @@
 
 <head>
 	<title>Login</title>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="stylesheet" type="text/css" href="./css/styles.css">
 </head>
 
 <body class="bg log">
+	<div class="menu_back"></div>
+	<div class="c_mobile_menu">
+		<ul>
+      <li><a href="/map">Map</a></li>
+			<li>News</li>
+			<ul class="submenu_mobile"}>
+        <li><a href="/blog">Blog</a></li>
+        <li><a href="/articles-publications">Articles and Publications</a></li>
+			</ul>
+			<li>How to</li>
+			<ul class="submenu_mobile"}>
+				<li><a href="/faq">FAQ</a></li>
+				<li><a href="/tutorials">Tutorials</a></li>
+				<li><a href="/definitions">Definitions</a></li>
+			</ul>
+      <li>About</li>
+			<ul class="submenu_mobile"}>
+				<li><a href="/the-project">The Project</a></li>
+				<li><a href="/partners">Partners</a></li>
+				<li><a href="/contact-us">Contact Us</a></li>
+			</ul>
+		</ul>
+		<a href="#" class="button_login">LOG IN</a>
+		<p class="forgot_text"><a href="#">Forgot your password?</a> <a href="#">Not a member</a></p>
+	</div>
 	<header class="menu_transparent">
+		<img class="icon_menu_mobile" id="open_menu" src="img/menu_icon.svg">
```

Comment:
Can you please add an `alt` attribute to all the images? Again, for accessibility and in case the images don't load.